### DISCUSSION TITLE
Discussing possible bugs

### DISCUSSION THREAD

**Discussion by kuv2707** [shivansh-bhatnagar18/multiplayer-uno#137] [General]
Use this page to bring to notice possible bugs

**Comment by sethdivyansh** [comment]
There are some bugs in `throwCards.ts`

- wild cards are not special type, they are wild type.
- On using the reverse card. Instead of changing the direction of the game immediately, it incorrectly allows the next player to take their turn before reversing the order.

correct me if I am wrong.
<img width="809" alt="image" src="https://github.com/shivansh-bhatnagar18/multiplayer-uno/assets/59174836/e311db51-26b8-45fc-9143-64962b60a451">

**Comment by kuv2707** [comment]
> wild cards are not special type, they are wild type.


This is intentional in anticipation that some rules require us separately treating the two.

>On using the reverse card. Instead of changing the direction of the game immediately, it incorrectly allows the next player to take their turn before reversing the order.


This is a legitmate concern - we'd have to move the game.nextPlayer at the top call into the individual cases.

You can put a PR for this
